script/generate-docs

```diff
@@ -19,4 +19,4 @@ STDERR.puts "Generating docs against #{remote}/#{branch}."
 # Explicitly use the guides/tsconfig.json file, because it includes
 # both the source and docs. The build itself just uses the root tsconfig,
 # which just hits src.
-`node node_modules/typedoc/bin/typedoc --tsconfig guides/tsconfig.json --options typedoc.json --gitRemote #{remote} --gitRevision #{branch}`
+`node node_modules/typedoc/bin/typedoc --tsconfig guides/tsconfig.json --gitRemote #{remote} --gitRevision #{branch}`
```

Comment:
For the library src folder does it recognize it automatically since you only specific the one from the guides folder here?

Reply:
The library src folder is picked from the typedoc options in the guides/tsconfig.json, we have specified two entry points "src/index.ts" and "guides/index.ts". Thats how it recognizes the two.